<launch>
  <arg name="robot_ip" default="192.168.3.127"/>
  <include file="$(find franka_control)/launch/franka_control.launch" pass_all_args="true"/>
  <arg name="arm_id" default="panda"/>
  <rosparam command="load" file="$(find franka_controllers)/config/controller_parameters.yaml" subst_value="true" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="LCQP_controller_multipleObstacle"/>
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_controllers)/launch/franka_with_obstacles.rviz -f $(arg arm_id)_link0 - -splash-screen $(find franka_visualization)/splash.png"/>
</launch>
